cmd/swarm/swarm-smoke/main.go

```diff
@@ -110,11 +110,10 @@ func main() {
 			Usage:       "whether to fetch content from a single node or from all nodes",
 			Destination: &single,
 		},
-		cli.IntFlag{
-			Name:        "track-timeout",
-			Value:       5,
-			Usage:       "timeout in seconds to wait for GetAllReferences to return",
-			Destination: &trackTimeout,
+		cli.BoolFlag{
+			Name:        "only-upload",
```

Comment:
don't we have `--single` already? why can't we reuse?

Reply:
`--single` means you will fetch from a single node, and not from all nodes.
`--only-upload` means that you will only upload and not fetch from anyone. As explained in the description of the PR, this is useful if you want to check where chunks have been synced without retrieving at all.

So `--single` and `--only-upload` are rather different.